Host bin should be used instead of the target bin (QT_INSTALL_BINS)
--- src/share/qtcreator/translations/translations.pro	2020-07-07 06:44:20.000000000 +0000
+++ src.mod/share/qtcreator/translations/translations.pro	2021-05-24 13:16:43.593579000 +0000
@@ -12,10 +12,10 @@
     return($$result)
 }
 
-XMLPATTERNS = $$shell_path($$[QT_INSTALL_BINS]/xmlpatterns)
-LUPDATE = $$shell_path($$[QT_INSTALL_BINS]/lupdate) -locations relative -no-ui-lines -no-sort
-LRELEASE = $$shell_path($$[QT_INSTALL_BINS]/lrelease)
-LCONVERT = $$shell_path($$[QT_INSTALL_BINS]/lconvert)
+XMLPATTERNS = $$shell_path($$[QT_HOST_BINS]/xmlpatterns)
+LUPDATE = $$shell_path($$[QT_HOST_BINS]/lupdate) -locations relative -no-ui-lines -no-sort
+LRELEASE = $$shell_path($$[QT_HOST_BINS]/lrelease)
+LCONVERT = $$shell_path($$[QT_HOST_BINS]/lconvert)
 
 wd = $$replace(IDE_SOURCE_TREE, /, $$QMAKE_DIR_SEP)
 
